### ISSUE TITLE
Can not reproduce the same results as the paper  

### ISSUE THREAD

**Issue by huihui43** [rimchang/RealBlur#7]
Hi, hope this finds you well. I find your work really awesome! I am recently trying to reproduce the results using the SRN-Deblur 's pre-trained model provided by your GitHub.  I follow the instruction from the "Test" suggested in your GitHub page:
`python run_model.py --phase=test --model=color --checkpoint_path=RealBlurJ_pre_trained+GOPRO+BSD500 --datalist=datalist/RealBlur_J_test_list.txt --height=784 --width=688`

The following is what I get 
![image](https://user-images.githubusercontent.com/29590745/130771735-2b9500ac-9c70-486f-a31f-0f6dbaba3d8b.png)
To my understanding, the psnr result I get is 29.60189 

Unfortunately, the result in the paper is 31.38

<img width="715" alt="Screenshot 2021-08-25 at 6 12 17 PM" src="https://user-images.githubusercontent.com/29590745/130772512-edfe5d2e-b133-4c4a-a343-19faf3b4ad31.png">

I have verified my code environment: Python 2.7, Cuda 8.0, open-cv 3.4.3.18, tf 1.4 & tf tensorboard 0.4, and I use the original random seed "0" from your code provided, but still, it seems that I cannot reproduce the same results of the paper.

I really appreciate it if you could point out something wrong regarding my experiments.  Thank you very much!

**Comment by rimchang** [comment]
We provide code for evaluation.

Please use below link for evaluation.

https://github.com/rimchang/RealBlur/blob/master/evaluation_RealBlur_ecc.py
